en/organisation/sammansattning/body.md

```diff
@@ -1,179 +1,100 @@
+The Vice chairman can be reached at [vordf@datasektionen.se](mailto:vordf@datasektionen.se).
 
-## The Treasurer - Fabian Andréasson
+## The Treasurer - Felix Murnion
 
 The Treasurer is responsible for the most things related to economics.
 This includes declarations, bills, payments, and accounting. The Treasurer
 is an authorized signatory for the chapter.
 
-__Contact__
+**Contact**
 
 The Treasurer can be reached at [kassor@datasektionen.se](mailto:kassor@datasektionen.se).
 
-## Vice Treasurer - Felix Murnion
-The Vice Treasurer assists the Treasurer with most things related to economics.
+## Vice treasurer
 
-__Contact__
+The Vice treasurer assists the Treasurer with most things related to economics.
 
-The Vice Treasurer can be reached at [vkassor@datasektionen.se](mailto:vkassor@datasektionen.se).
+**Contact**
 
-## The Secretary - Herman Karlsson
+The Vice treasurer can be reached at [vkassor@datasektionen.se](mailto:vkassor@datasektionen.se).
 
-The Secretary is responsible for documenting D-directorate and Chapter Meetings
+## The Secretary
+
+The Secretary is responsible for documenting D-directorate and chapter meetings
 and to write protocols and archive them.
 
-__Contact__
+**Contact**
 
 The Secretary can be reached at [sekreterare@datasektionen.se](mailto:sekreterare@datasektionen.se).
 
-__Contact person to__
-
-* Nominations committee chairperson
-
-* Nominations committee members
+## Board member of Business Relations - Adam Sjöberg
 
-* Student union assembly delegates
-
-## Board Member of Educational Affairs - Abhinav Sasikumar
-
-This board member works closely with the study council and works as a
-link between them and the board.
-
-__Contact__
-
-The Board Member of Educational Affairs can be reached at [d-uf@datasektionen.se](mailto:d-uf@datasektionen.se).
-
-__Contact person to__
-
-* Program responsible student
-
-* The Chairman of the Study Council
-
-* The Chairman of the Equality Committee
-
-* International Student Coordinator
+Works with business related question and our projects and committees related to this area.
+Is also responsible for internal communications and that important information is available to our members.
 
-* Student Safety Officer
+**Contact**
 
-* Project Leader for Project Pride
+The Board member of Business Relations can be reached at [d-nok@datasektionen.se](mailto:d-nok@datasektionen.se).
 
 ## Board member of Student Issues - Malin Svenberg
 
 Works with question regarding the well being of our members, working environment, and
 social events.
 
-__Contact__
+**Contact**
 
 The Board member of Student Issues can be reached at [d-ssf@datasektionen.se](mailto:d-ssf@datasektionen.se).
 
-__Contact person to__
-
-* Head of the Bar
-
-* Chapter Sports Leader
+## Board member of Educational Affairs - Abhinav Sasikumar
 
-* Light and Sound responsible 
-
-* Chief of Halls
-
-* Cultural Attaché
-
-* Prylis
-
-* Mulle/Mullerina Schmeck
-
-* DESCtop
-
-* Ärkedemon
```

Comment:
If all of this is removed, have you made it explicit somewhere else?

Reply:
Yes, I added a link to an external document on line 8, [dsekt.se/kontaktfunktionarare](dsekt.se/kontaktfunktionarare).

Reply:
On second thought the link should probably be written from the perspective of the elected officials and not D-rek...

Reply:
Updated to [dsekt.se/kontaktperson](https://dsekt.se/kontaktperson).